forum/qa-plugin/block-pw/block-pm-user-list-page.php

```diff
@@ -17,16 +18,16 @@ public function match_request(string $request): bool
         return $request === 'blocked-users';
     }
 
-    public function process_request(string $request): ?array
+    public function process_request(): ?array
```

Comment:
czyli nie czujesz potrzeby weryfikacji, czy przychodzi poprawny request?

Reply:
Jeśli dobrze rozumiem filozofię q2a, linijkę wyżej sprawdzam czy to jest request który mnie interesuje - `return $request === 'blocked-users';`. Jeśli odpala się `process_request()` to znaczy, że `match_request()` zwróciło `true` więc ten request chce przechwycić - sprawdzanie w kolejnej funkcji nie ma sensu.